thirtyfour/src/webelement.rs

```diff
@@ -782,6 +783,47 @@ impl WebElement {
             .await?;
         Ok(())
     }
+
+    /// Get the parent of the WebElement.
+    ///
+    /// # Example
+    /// ```no_run
+    /// # use thirtyfour::prelude::*;
+    /// # use thirtyfour::support::block_on;
+    /// #
+    /// # fn main() -> WebDriverResult<()> {
+    /// #     block_on(async {
+    /// #         let caps = DesiredCapabilities::chrome();
+    /// #         let driver = WebDriver::new("http://localhost:4444", caps).await?;
+    /// let elem = driver.find(By::Id("child")).await?;
+    /// let parent = elem.get_parent().await?;
+    /// #         driver.quit().await?;
+    /// #         Ok(())
+    /// #     })
+    /// # }
+    /// ```
+    ///
+    /// Note: This helper performs a new query for the parent element,
+    /// with the sessions current settings. If you need more control,
+    /// start with the equivalent query and customize to your needs.
```

Comment:
This seems potentially confusing to me and I think it's probably fine just to remove this whole note and the subsequent example. It's not obvious in this context why someone might need more control or customization. It immediately causes the reader to think "wait, is there something else I should know here?" or "should I be using something else instead?".

Generally it's good to reduce the number of webdriver requests and combine things into a single query where possible, but this probably isn't the best place for that level of documentation.